### DISCUSSION TITLE
ALT/AZ telescope start position in Stellarium

### DISCUSSION THREAD

**Discussion by verbalistique** [Stellarium/stellarium#2388] [Q&A] [answered]
Good morning all !

My telescope is set up on an alt/azimuth fork mount (a MEADE LX90 GOTO with handbox/remote #497).
I use Stellarium in the following way:
I start Stellarium first, then I connect the USB/serial cable to the PC, then I turn on the telescope that I have previously set up (level tripod, level tube and oriented due North), and I validate on the hand control the requested information (date, time, time difference).

Stellarium detects the telescope well (thanks to the "telescope control" plugins) and displays the orange crosshair corresponding to its relative position.
However the position of the orange reticle which should point due North, 0° is offset to the position of the telescope during the previous session. (even if I close Stellarium and even if the telescope has been turned off). (For the rest it works, if I point to a star and I ask Stellarium to point the telescope towards it, the telescope moves)

I also tried stopping the plugins before turning on the telescope and then restarting it, but that didn't change anything.

Is this normal?
Is there a possibility to ask to Stellarium to resume the "PARK" position at 0° and due NORTH at the start of the session?
One last question which is certainly related: Can we get Stellarium to display a virtual handbox/remote for the LX90 (like the one found with ASCOM)?

Hoping to be in the right section,
Looking forward to,
Cordially.
Charles

**Comment by gzotti** [comment, marked as answer]
Stellarium shows the reticle on the screen on the position which the telescope reports. Stellarium then sends "GOTO RA/DEC" commands when you press "slew to selected". It reads as if the telescope does not know where it is pointing.
Can you go to objects using the hand controller alone in this setup? 
Stellarium has no awareness of PARKing. This is left to the telescope's own drive. 

If you drive your telescope as ASCOM telescope, you should get everything that ASCOM provides. In Stellarium's sources there is no virtual handbox GUI.

**Comment by verbalistique** [comment]
Hello Georg and thank's for answer!

I try tonight and Said to you!
With the remote, the télescope is fully ok, goto and navigation !

Le jeu. 7 avr. 2022 à 14:19, Georg Zotti ***@***.***> a
écrit :

**Comment by verbalistique** [comment]
Indeed I found where the problem came from thanks to your explanation !!! **Thank you** In fact on the MEADE #497 hand controllers, at start-up, you have to enter the date, the time and then the time difference. Then the remote asks which method we want to use for precision alignment (easy, one star or two stars). After having chosen one of these three methods, a message appears on the remote control, explaining that it is necessary to put the telescope in station (level tripod, level tube and full North). It is therefore necessary to validate on the remote control (ENTER) to then have the choice of the stars to be selected for the alignment. It is only once the message has been validated that the remote control sends its placement coordinates to Stellarium!!! (actually I stopped just before this message, thinking that I could do the alignment via Stellarium) So problem solved!!!! **many thanks Gzotti!!!** If, on the other hand, someone knows of a MEADE virtual remote control that works under Windows 10, I'm interested because Meade is apparently not ASCOM compatible.

Best regards !! Bien a tous !!
Charles

Effectivement j'ai trouvé d'où venait le problème grâce à votre explication !!! **Merci**
En fait sur les raquettes de commande MEADE #497, au démarrage, il faut renseigner la date, l'heure puis le décalage horaire. Ensuite la télécommande demande quelle méthode nous voulons utiliser pour l'alignement de précision (facile, une étoile pou deux étoiles). Apres avoir choisi une de ces trois méthodes, un message apparait sur la télécommande , expliquant qu'il faut mettre le télescope en station (trépied de niveau, tube de niveau et plein Nord). Il faut donc valider sur la télécommande (ENTER) pour ensuite avoir le choix des étoiles a sélectionner pour l'alignement. Ce n'est qu'une fois le message validé que la télécommande envois ses coordonnées de placement a Stellarium !!! (en fait je m'arrêtais juste avant ce message, pensant que je pouvait faire l'alignement via Stellarium) Donc problème résolu !!!! **un grand merci Gzotti !!!**
Si par contre quelqu'un connait une télécommande virtuelle MEADE qui fonctionne sous Windows 10, ca m'intéresse car Meade n'est pas compatible ASCOM apparemment.
